CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for ui-users
 
+## [11.1.0](https://github.com/folio-org/ui-users/tree/v11.1.0) ()
+[Full Changelog](https://github.com/folio-org/ui-users/compare/v11.1.0...v11.1.0)
+* Update permissions for mod-patron. Ref UIU-3259
+
```

Comment:
Hey @azizbekxm 
This is not the correct way of updating the changelog in your case.

Reply:
thanks for suggestion, fixed this